[obv] testsuite: Fix gdb.arch/i386-bp_permanent.exp regression
http://sourceware.org/ml/gdb-patches/2010-01/msg00554.html
http://sourceware.org/ml/gdb-cvs/2010-01/msg00203.html

### src/gdb/testsuite/ChangeLog	2010/01/22 17:06:49	1.2099
### src/gdb/testsuite/ChangeLog	2010/01/24 04:29:32	1.2100
## -1,3 +1,8 @@
+2010-01-24  Jan Kratochvil  <jan.kratochvil@redhat.com>
+
+	* gdb.arch/i386-bp_permanent.exp (Disassemble function '$function'):
+	Adjust it for DISASSEMBLY_OMIT_FNAME.
+
 2010-01-22  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	PR symtab/11199:
--- src/gdb/testsuite/gdb.arch/i386-bp_permanent.exp	2010/01/01 07:32:00	1.3
+++ src/gdb/testsuite/gdb.arch/i386-bp_permanent.exp	2010/01/24 04:29:32	1.4
@@ -62,7 +62,7 @@
 set function standard
 
 set retcode [gdb_test_multiple "disassemble $function" "Disassemble function '$function'" {
-    -re ".*($hex) <$function\\+0>.*($hex) <$function\\+4>.*($hex) <$function\\+5>.*($hex) <$function\\+6>.*$gdb_prompt $" {
+    -re ".*($hex) <\\+0>.*($hex) <\\+4>.*($hex) <\\+5>.*($hex) <\\+6>.*$gdb_prompt $" {
       set function_start $expect_out(1,string);
       set address $expect_out(2,string);
       set address1 $expect_out(3,string);
